docs/guide/http_cache.md

```diff
@@ -607,6 +607,17 @@ Note that the system does not add this tag to responses itself, just purges if p
 Response tagging using this tag is currently meant to be done inline in the template logic / views
 based on your decision.
 
+!!! tip Troubleshooting - Content tagged by a big number of tags (too long headers)
+    
+    In case of complex Content, for instance Landing Pages with many blocks, you might get into trouble with too long response `xkey` header. Because of this, necessary cache entries may not be tagged properly. You will see `502 Headers too long` errors then.
```

Comment:
👍 

Could be adapted to something like:
> In case of complex Content, for instance Landing Pages with many blocks, you might get into trouble with too long response `xkey` header. Because of this, necessary cache entries may not be tagged properly, and you'll see `502 Headers too long` errors.
> If this is the case, please customize the following runtime settings on your Varnish instance(s):
> 
>     - [http-resp-hdr-len](https://varnish-cache.org/docs/5.1/reference/varnishd.html#http-resp-hdr-len) _(e.g. 32k)_
>    - [http_req_hdr_len](https://varnish-cache.org/docs/5.1/reference/varnishd.html#http-max-hdr) _(e.g. 64k)_
 >   - [http-resp-size](https://varnish-cache.org/docs/5.1/reference/varnishd.html#http-resp-size) _(e.g. 16k)_

Reply:
Please adjust the example values to something you find sensible, this was only a suggestion for guidline.